cmd/loop/loopin.go

```diff
@@ -32,9 +32,16 @@ var (
 			"with our reserved prefix: %v.",
 			labels.MaxLength, labels.Reserved),
 	}
-	routeHints = cli.StringFlag{
-		Name:  "route_hints",
-		Usage: "",
+	routeHintsFlag = cli.StringSliceFlag{
+		Name: "route_hints",
+		Usage: "Route hints that can each be individually used " +
+			"to assist in reaching the invoice's destination.",
+	}
+	privateFlag = cli.BoolFlag{
+		Name: "private",
+		Usage: "Indicates the offchain destination is private, " +
+			"or only reachable via private channels. Route hints " +
+			"will be automatically included",
```

Comment:
Comment still applies? I don't think end-users need to understand the concept of "private destination", just that this will add hop hints + expose their private chans